node/chainSimulator/chainSimulator.go

```diff
@@ -129,30 +124,23 @@ func computeStartTimeBaseOnInitialRound(args ArgsChainSimulator) int64 {
 }
 
 func (s *simulator) createTestNode(
-	configs *config.Configs,
-	shardIDStr string,
-	gasScheduleFilename string,
-	apiInterface components.APIConfigurator,
-	bypassTxSignatureCheck bool,
-	initialRound int64,
-	minNodesPerShard uint32,
-	minNodesMeta uint32,
+	outputConfigs *configs.ArgsConfigsSimulator, args ArgsChainSimulator, shardIDStr string,
 ) (process.NodeHandler, error) {
```

Comment:
for the next PR, we should return the output configs as an output parameter in order to avoid working on the same pointer